dist/css/ray.css

```diff
@@ -6963,19 +6971,6 @@ button.bg-linkedin:focus {
   border-radius: 50%;
   width: 60%;
   height: 60%; }
-  .avatar-provider-lg {
-    width: 75%;
-    height: 75%;
-    right: -30%;
-    bottom: -30%; }
-  .avatar-provider-md {
-    width: 60%;
-    height: 60%;
-    right: -10%;
-    bottom: -10%; }
-  .avatar-provider-sm {
-    width: 40%;
-    height: 40%; }
```

Comment:
I don't understand why you removed the positions? It solves our problem but the render is not right anymore
![image](https://user-images.githubusercontent.com/29859169/67217764-32802a80-f41d-11e9-88e8-7785ace4bf27.png)

Reply:
I wanted to simplify it and then ask a designer exactly what they're looking for.  Not finished with it...

Reply:
Ok sorry. Danielle gave me these position values. You should ask her if you want.

Reply:
Ah ok I didn't realise.  I'll try to make it work with these then.